solutions/hw3/task_overview.md

```diff
@@ -0,0 +1,5 @@
+# Homework 3
+### Preparing data
+### Analyze
+### Visualize
```

Comment:
When using seaborn, your plot prints `<AxesSubplot:xlabel='None-Sex', ylabel='Pclass'>`. We discussed this in the class, you need to import `pyplot` and use `plt.show()`, because `seaborn` is just an extension of `matplotlib`.

Reply:
Also, I am not sure about the heatmap. Why is the X axis called `None-Sex`? :) 

Please add some legend or title so that it's obvious that these are probabilities.